ando/tests/test_engine.py

```diff
@@ -45,7 +45,7 @@ def test_AnDO_Func_subject_level(self):
         names = ['exp-Landing', 'anye', 'sess-180116_001_m_anye_land-001',
                  'source']
         validate = list()
-        self.assertEqual(all(andoE.is_AnDO_R(names, 0, validate)), False)
+        self.assertEqual((andoE.is_AnDO(names,False)), False)
```

Comment:
idem... it should be (names, False) with a space...

Reply:
Fixed